api/src/main/java/io/opencensus/trace/Tracing.java

```diff
@@ -82,6 +82,14 @@ static TraceComponent loadTraceComponent(ClassLoader classLoader) {
           Class.forName("io.opencensus.trace.TraceComponentImpl", true, classLoader),
           TraceComponent.class);
     } catch (ClassNotFoundException e) {
+      logger.log(Level.FINE, "Try to load lite implementation.", e);
```

Comment:
Should this message give more detail, like "Couldn't load full implementation for TraceComponent, now trying to load lite implementation."?